pkg/docker/docker_test.go

```diff
@@ -0,0 +1,212 @@
+package docker_test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+
+	"github.com/testground/testground/pkg/docker"
+	"github.com/testground/testground/pkg/rpc/rpctest"
+)
+
+var (
+	cli *client.Client
+)
+
+func init() {
+	var err error
+	cli, err = client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		panic(err)
+	}
+
+	cli.NegotiateAPIVersion(context.Background())
+	rand.Seed(time.Now().UnixNano())
+}
+
+func errfail(t *testing.T, err error) {
```

Comment:
Nitpick, but I don't think we need this.

Just having:
```
if err != nil {
  t.Fatal(err)
}
```

is fine.

---

Also without a call to `t.Helper()` I don't think this will be reporting correct line numbers on where the error happened, so it'd be hard to detect what error happened if a test calls it >1 times.

Reply:
Just use `require.NoError`. Look at `TestAllEvents` in the SDK, or at `selector_test.go`.